xAPI.md

```diff
@@ -186,59 +186,70 @@ learning that SCORM could not enable.
-*	3.0.	[Definitions](#definitions)  
-*	4.0.	[Statement](#statement)  
-    *	4.1.	[Statement Properties](#stmtprops)  
-        *	4.1.1.	[ID](#stmtid)  
-        *	4.1.2.	[Actor](#actor)  
-        *	4.1.3.	[Verb](#verb)  
-        *	4.1.4.	[Object](#object)  
-        *	4.1.5.	[Result](#result)  
-        *	4.1.6.	[Context](#context)  
-        *	4.1.7.	[Timestamp](#timestamp)  
-        *	4.1.8.	[Stored](#stored)  
-        *	4.1.9.	[Authority](#authority)  
-        *	4.1.10.	[Version](#version)  
-        *	4.1.11.	[Attachments](#attachments)  
-        *	4.1.12.	[Data Constraints](#dataconstraints)  
-    *	4.2.	[Retrieval of Statements](#retstmts)  
-	*	4.3.	[Voided](#voided)  
-	*	4.4.	[Signed Statements](#signature)  
-*	5.0.	[Miscellaneous Types](#misctypes)  
-    *	5.1.	[Document](#miscdocument)  
-    *	5.2.	[Language Map](#misclangmap)  
-    *	5.3.	[Extensions](#miscext)  
-    *	5.4.	[Identifier Metadata](#miscmeta)  
-*	6.0.	[Run-time Communication](#rtcom)  
-    *	6.1.	[Encoding](#encoding)  
-    *	6.2.	[API Versioning](#apiversioning)  
-    *	6.3.	[Concurrency](#concurrency)  
-    *	6.4.	[Security](#security)  
-		*	6.4.1.	[OAuth 1.0 Authentication Scenarios and Methods](#authdefs)  
-		*	6.4.2.	[OAuth 1.0 Authorization Scope](#oauthscope)  
-*	7.0.	[Data Transfer (REST)](#datatransfer)  
-    *	7.1.	[Error Codes](#errorcodes)  
-    *	7.2.	[Header Parameters](#header-parameters)  
-    *	7.3.	[Statement API](#stmtapi)  
-    	*	7.3.1. [PUT Statements](#stmtapiput)  
-    	*	7.3.2. [POST Statements](#stmtapipost)  
-    	*	7.3.3. [GET Statements](#stmtapiget)  
-    	*	7.3.4. [Voided Statements](#voidedStatements)  
-    *	7.4.	[Document APIs](#docapis)  
-    *	7.5.	[State API](#stateapi)  
-    *	7.6.	[Activity Profile API](#actprofapi)  
-    *	7.7.	[Agent Profile API](#agentprofapi)  
-    *	7.8.	[About resource](#aboutresource)  
-    *	7.9.	[Alternate Request Syntax](#alt-request-syntax)  
-    *	7.10.	[Validation](#validation)  
-    *	7.11.	[HTTP HEAD](#httphead)  
+*	Part One:	[Summary of the Experience API](#partone)  
+	*	1.0.	[Role of the Experience API](#roleofxapi) 
+	*	2.0.	[How To Use This Document](#readingguidelines)  
+		*	2.1.	[MUST / SHOULD / MAY](#def-must-should-may)  
+	 	*	2.2.	[Guidelines for Interpreting Descriptive Text and Tables](#interpret-text-table)  
+	*	3.0.	[Serialization and JavaScript Object Notation](#json)
+	*	4.0.	[Definitions](#definitions) 
+	*	5.0.	[xAPI Components](#extending-xapi) 
+	*	6.0.	[Extending xAPI](#roleofxapi) 
+	*	7.0.	[Profiles and Communities of Practice](#COPs)  
+*	Part Two:	[Experience API (xAPI) Data](#parttwo)  
+	*	1.0.	[Documents](#documents) 
+	*	2.0.	[Statements](#statement)  
+		*	2.1.	[Purpose](#statement-purpose)  
+	 	*	2.2.	[Formatting Requirements](#dataconstraints) 
+	 	*	2.3.	[Statement Lifecycle](#lifecycle) 
+		 	*	2.3.1.	[Statement Immutability](#statement-immutablity-and-exceptions) 
+		 	*	2.3.2.	[Voiding](#voided) 
```

Comment:
Links to wrong place (maybe we have an extra anchor in the middle of statement lifecycle?)